### DISCUSSION TITLE
Can social bookmarking website like reddit be done with Directus?

### DISCUSSION THREAD

**Discussion by jayabie** [directus/directus#16477] [Archive]
Hello Team,

I have installed Directus on a local machine successfully, got to the admin panel.
but I'm not a technical guy, i want to achieve something like reddit, i know reddit is big, something simple with basic functionally.
can this be achieved with Directus? if yes, can you please tell me how by providing some resources?

**Comment by rijkvanzanten** [comment]
Should be possible! The main challenge would be figuring out the ideal data model for such a setup. I imagine you'd have to start with three main collections to have the MVP: "subreddits" (channels/groups/whatever you wanna call it), "posts", and "users". Each post is associated to a group, and each post can have a "parent post" (that's how you'd achieve nested threads). Each post also has an author (the user that posted it). With those basic collections and relations, I believe you already get most of the way there with a reddit style message board 🙂

**Comment by vitalijalbu** [comment]
You can do anything, if need help dm me.
